components/cards.html

```diff
@@ -0,0 +1,122 @@
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/style-docs.css" rel="stylesheet" />
+    <link rel="shortcut icon" type="image/jpg" href="/assets/filled-leaf-logo.png">
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
+    <title>mint | component library</title>
+</head>
+<body>
+
+        <header>
+            <div class="align">
+                <div class="nav-logo">
+                    <a href="/index.html"><img src="/assets/filled-leaf-logo.png" class="logo-nav"></a>
+                    <a href="/index.html" class="link-style"><h2>mint</h2>
+                 </a>
+                 </div> 
+                 <div class="nav-top hidden-sm">
+                    <div class="nav-float-right"><a href = "/index.html" class="hover-effect">home</a></div>
+                    <div class="nav-float-right"><a href = "/documentation.html" class="hover-effect">documentation</a></div>
+                    <div class="nav-float-right"><a href = "https://github.com/sneha-pujari/Component-Library" class="hover-effect">github</a></div>
+                </div>
+    
+            </div>
+        </div>
+        </header>  
+            <nav>
+                <h2>Getting started</h2>
+                <ul>
+                  <li><a class="link-style" href="/documentation.html">Introduction</a></li>
+                </ul>
+                <h2>Components</h2>
+                <ul>
+                    <li><a class="link-style" href="avatar.html">Avatar</a></li>
+                    <li><a class="link-style" href="alerts.html">Alerts</a></li>
+                    <li><a class="link-style" href="badges.html">Badges</a></li>
+                    <li><a class="link-style" href="buttons.html">Buttons</a></li>
+                    <li><a class="link-style" href="cards.html">Cards</a></li>
+                    <li><a class="link-style" href="images.html">Images</a></li>
+                    <li><a class="link-style" href="input.html">Input</a></li>
+                    <li><a class="link-style" href="list.html">List</a></li>
+<!--                     <li><a class="link-style" href="typography.html">Typography</a></li> -->
+                </ul>
+              </nav>
+              <main>
+                <h2 style="font-size:35px">Cards</h2>
+                <section>
+                        <h3>Card Vertical</h3>
+                        <div class="card-container">
+                            <img class="card-img" src="https://bit.ly/2Om0Udn"> 
+                            <h3>Card Heading</h3>
+                            <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an un</p>
+                            <button class="primary-button">Add to cart</button>

```

Comment:
Button is positoned more left than other elements. Make it one? IMO, would look better